red/runtime/nodes/context/index.js

```diff
@@ -268,7 +268,29 @@ function createContext(id,seed) {
             }
             context = getContextStorage(storage);
         }
-        context.set(scope, key, value, callback);
+        if (!Array.isArray(key) || !Array.isArray(value) || key.length !== value.length) {
```

Comment:
If they provide two arrays of different lengths, this will use the first array to be used as the key name - which is invalid.

If the key is an Array, it should go to the `else` clause here - regardless of length. If the `value` array is shorter than the `key` array, use `null` for missing values. If the `value` array is longer, then you just ignore the extra values.

Reply:
Go with option 1 - use the one value for key[0] and null for the rest.

Reply:
Thank you for feedback.
I fixed this.